packages/subgraph/schema.graphql

```diff
@@ -1196,6 +1196,12 @@ type AccountTokenSnapshot @entity {
     updatedAtTimestamp: BigInt!
     updatedAtBlockNumber: BigInt!
     # ---------------------------------- state ----------------------------------
+
+    """
+    isLiquidationEstimateOptimistic, If they have `0` subcription with no units it return true or else false.
```

Comment:
I think currently the logic is the other way around, if subscriptionWithUnits > 0, it is true. This is fine and I think it makes sense especially since we want the default value to be false.
It would be good to mention here that optimistic means that it is the earliest time the user will be liquidated as they may receive ongoing distributions which aren't tracked by the subgraph.
Also small typo: `subcription` => `subscription`